Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -214,9 +214,11 @@ def find_indicators_to_limit_loop(indicator_query: str, limit: int, total_fetche
         (tuple): The iocs and the last page
     """
     iocs: List[dict] = []
+    if last_found_len is None:
+        last_found_len = PAGE_SIZE
```

Comment:
why did you switch from using a default value?

Reply:
Because `PAGE_SIZE` value can change in `main`, so this will create a different `PAGE_SIZE` than the one set by the `default value` when the function was defined.
e.g.
1. PAGE_SIZE init value was 2000
2. PAGE_SIZE was changed to 100 in main
3. The default value for `last_found_len` is 2000.

Reply:
cool